server/admin_management/api/app/routers/router_access_control_privilege.py

```diff
@@ -99,3 +105,23 @@ def create_access_control_privilege_many(
             )
 
         audit_event_log.log_event()
+
+
+@router.get(
+    "/{application_id}/access_control_privileges",
```

Comment:
Hi Catherine, we probably like to move this endpoint to other router (application) or at least rename the base-path and sub-path for better sticking to the **REST standard** as this does not follow...
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/f2d76c4b-97ff-4531-ac81-96a8f2ecd6ed)
I myself find in this case is also blur to what REST should be followed...

Some suggestions:
These two probably are more follow from standard.
```
GET: /applications/{application_id}/delegated-admins/privileges
GET: /access_control_privileges?{application_id}
```

These two I am not sure myself if they can are acceptable:
```
GET: /access_control_privileges/applications/{application_id}/privileges
GET: /delegated-admin-management/applications/{application_id}/privileges
```
Maybe @basilv can provide some good ideas? Thanks!

Reply:
we might still want to keep it under this router, cause the return result is a list of users with roles, not users and applications. I feel it makes sense to keep under this router. But we could rename it, `access_control_privileges/{application_id}/privileges`? I also think about the path before but has no good idea....

Reply:
By standard, if `access_control_privileges` is the **REST "resource"**, then the path **"/{application_id}"** isn't correct, that would indicate the "**id**" to identify that REST **resource**. (Just like on the network tab, you will only see "/access_control_privileges/**19**/" and that "19" isn't (`application_id`) for others ( it is the resource for `access_control_privileges.get(id=19)` ).

Reply:
`access_control_privileges/privileges/?application_id={application_id}` ?

Reply:
discussed with team, use `/access_control_privileges?application_id={application_id}` and `/application_admins?application_id={application_id}`

Reply:
![image](https://github.com/bcgov/nr-forests-access-management/assets/23131735/02626eec-0bd9-430a-bc7c-30f3da68a786)

Reply:
There's one more level of potential debate in terms of the actual resource naming. /application_admins isn't actually returning a list of admin users, its returning the access held by these users. So perhaps /application_admin_accesses

/access_control_privileges reflects access, not users so that's more in line, but could be more clearly named /delegated_admin_accesses to better align with our domain language.

Reply:
I feel the same, "delegated_admin_accesses" is a better name, and the existing endpoints was inherited from the past using python database entity name, with not really a naming convention followed from domain language.
However, probably up to Catherine if she wants to change or leave it to the new ticket to change them all together (will impact frontend).